src/OpenLoco/src/ScenarioManager.cpp

```diff
@@ -380,18 +380,11 @@ namespace OpenLoco::ScenarioManager
         }
 
         const auto currentState = getCurrentScenarioFolderState();
-
-        if (!tryLoadIndex(currentState))
+        if (!tryLoadIndex(currentState) || forceReload)
```

Comment:
I suppose it would be a small optimisation to instead use:
```suggestion
        if (forceReload || !tryLoadIndex(currentState))
```
Shall we?